### PR TITLE
Remove legacy GQL queries not used anymore

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: test/graphql/queries/benchmark_query_test.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@
 class BenchmarkQueryTest < ActiveSupport::TestCase
   test 'query benchmark owned by the user' do
     query = <<-GRAPHQL
-      query allBenchmarks {
-          allBenchmarks {
+      query latestBenchmarks {
+          latestBenchmarks {
```
Comment (akofink) [inline test/graphql/queries/benchmark_query_test.rb]: Fine for this PR, but this has me thinking that we should match the TestResult API and allow `bool latest` as an argument.

File: app/graphql/types/query.rb [hunk 2]
```diff
@@ -23,32 +23,15 @@ def collection_field(name, type)
...
       description 'Details for a system'
       argument :id, String, required: true
     end
 
-    field :all_image_streams, [Types::System], null: true do
-      description 'All image streams visible by the user'
-      argument :search, String, 'Search query', required: false
-    end
-
     field :all_profiles, [Types::Profile], null: true do
```
Comment (akofink) [inline app/graphql/types/query.rb]: This one will be removed soon as well, right?

File: test/graphql/queries/benchmark_query_test.rb (hunk 1 above)
Comment (dLobatog) [reply test/graphql/queries/benchmark_query_test.rb]: Agreed - ideally all fields can be defined as `collection_field :systems, Types::System` - and just parameterized via the "search" field

File: app/graphql/types/query.rb (hunk 2 above)
Comment (dLobatog) [reply app/graphql/types/query.rb]: Yup, I've seen it used on 1 place still though

**Pull request merged**